charts/atlantis/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: v0.22.3
 description: A Helm chart for Atlantis https://www.runatlantis.io
 name: atlantis
-version: 4.10.3
+version: 4.10.4
```

Comment:
```suggestion
version: 4.11.0
```

Reply:
version has been upgraded as per the suggestion.